config.json

```diff
@@ -18,6 +18,17 @@
 	    ]
 	},
 	"propagatedmount": "/mnt",
+	"mounts": [
+		{
+		"name": "KeyPath",
+		"description": "Path to SSH keys",
+		"source": "/tmp",
+		"destination": "/tmp/sshfs",
```

Comment:
The current design allows the user to create multiple volumes using different keys, so long as the keys are all in the same directory.

Changing the plugin to take the path to a single key would effectively limit the user to working with remote hosts that all share the same key at any one time.

This is because changing the setting to accept a different path requires stopping any containers that are using the volume created by the plugin, and removing those volumes.